app/controllers/follow_ups_controller.rb

```diff
@@ -7,13 +7,11 @@ def create
     @follow_up = FollowUp.new(**follow_up_params)
     authorize @follow_up
     if save_follow_up.success?
-      respond_to do |format|
-        # html is used for the show page
-        format.html do
-          redirect_to(structure_user_follow_ups_path(@user.id, anchor:))
-        end
-        format.turbo_stream { replace_new_button_cell_by_follow_up_status_cell } # turbo is used for index page
-      end
+      # when coming from the follow_ups index page
+      return redirect_to structure_user_follow_ups_path(@user.id) if request.referer.include?("/follow_ups")
+
+      # when coming from the all_users_table index page
+      replace_new_button_cell_by_follow_up_status_cell
```

Comment:
On a plus besoin d'utiliser ce turbo_stream ici aussi je pense, avec le morphing la cellule va être remplacé automatiquement sur la page index. Du coup je pense qu'on peut juste faire
```suggestion
      redirect_to request.referer
```
ce qui est top parce que ce controller n'a pas à savoir d'où est-ce qu'il est appelé